### ISSUE TITLE
Hit Sound Adjustment

### ISSUE THREAD

**Issue by da11as911** [Tripperful/sf-servers-requests#13]
Not sure what all is possible here, but something along the lines of:
1) If the player is in the line of sight for a non-hitscan damage then allow hitsound, otherwise no sounds will be played for non hitscan weapons. This seems to make the most sense and will not allow for a competitive advantage of player seeking in a game.
2) Also the implementation of different pitch sounds depending upon damage amount that New Guy spoke about in Discord
3) Client ability to have hitsounds = 0, 1, 2 (Off, Hitscan Only, All)

(request)

**Comment by Tripperful** [comment]
Non-hitscan hitsounds (1) implemented, pitch variation (2) is still pending.
(3) Will not be implemented, this shouldn't be a client setting, the client would prefer as much info as the server allows anyways

**Comment by da11as911** [comment]
Don't you have the ability to turn hit sounds off via the menu or am I mistaken? Also - I tried changing the hitsounds the other day from SF  to a default and it didnt work. Is this a bug or just me? I'll do some more testing and report back.

**Comment by da11as911** [comment]
Putting this here for reference since it was archived on discord, this is New Guys pitch idea from the HL2DM Competitive discord:

```
Damage: 01-07
Wave       = 00_quick
Volume     = 5
Pitch      = 1.20

Damage: 08-14
Wave       = 00_quick
Volume     = 7
Pitch      = 1.00

Damage: 15-23
Wave       = 00_quick
Volume     = 9
Pitch      = 0.80

Damage: 24-32
Wave       = 01_low
Volume     = 5
Pitch      = 1.20

Damage: 33-49
Wave       = 01_low
Volume     = 7
Pitch      = 1.00

Damage: 50-74
Wave       = 01_low
Volume     = 9
Pitch      = 0.80

Damage: 75-99
Wave       = 02_medium
Volume     = 5
Pitch      = 1.20

Damage: 100-124
Wave       = 02_medium
Volume     = 7
Pitch      = 1.00

Damage: 125-149
Wave       = 02_medium
Volume     = 9
Pitch      = 0.80

Damage: 150-174
Wave       = 03_high
Volume     = 5
Pitch      = 1.20

Damage: 175-199
Wave       = 03_high
Volume     = 7
Pitch      = 1.00

Damage: 200+
Wave       = 03_high
Volume     = 9
Pitch      = 0.80
```
